jay-config/src/_private/ipc.rs

```diff
@@ -256,6 +256,7 @@ pub enum ClientMessage<'a> {
     ShowWorkspace {
         seat: Seat,
         workspace: Workspace,
+        move_pointer: bool,
     },
```

Comment:
You cannot modify messages because that would break existing config.sos. See https://github.com/mahkoh/jay/blob/f94f199ab12a4545a455a04aec11ec84b62f7c27/jay-config/src/_private/client.rs#L1053-L1066 for how to handle this.